runtime/src/main/scala/akka/grpc/internal/ServerReflectionImpl.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.internal
+
+import com.google.protobuf.Descriptors.FileDescriptor
+
+import akka.NotUsed
+import akka.stream.scaladsl._
+
+import _root_.grpc.reflection.v1alpha.reflection._
+
+class ServerReflectionImpl private (fileDescriptors: Map[String, FileDescriptor], services: List[String])
```

Comment:
`@InternalApi`and `private[akka]` etc. perhaps `final` as well?

Reply:
yeah - that's mostly implicit from being in the `akka.grpc.internal` package, but good to make more obvious